### PR TITLE
chore: create `dependabot.yml`

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
This will create automatic PRs to update both dependencies + actions used in workflows, so we won't need manual PRs like #298 anymore

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
+
+  - package-ecosystem: npm
+    directory: /
+    schedule:
+      interval: daily
```
Comment (bmish) [inline .github/dependabot.yml:11]: Can we do `weekly`? I'm a big fan of dependabot but I don't want to get spammed by updates every day.

```suggestion
      interval: weekly
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
```
Comment (bmish) [inline .github/dependabot.yml:6]: Can we do `weekly`? I'm a big fan of dependabot but I don't want to get spammed by updates every day.

```suggestion
      interval: weekly
```

Comment (MichaelDeBoey) [reply .github/dependabot.yml:6]: Since we're only using major version on actions, I think it would be better to keep `daily`, so we can take action asap

Comment (bmish) [reply .github/dependabot.yml:6]: Fair enough.

**Review by bmish — APPROVED** [review]

**Pull request merged**